js/screen-saver.js

```diff
@@ -0,0 +1,46 @@
+var app = app || {};
+
+app.screenSaver = (function() {
+
+	'use strict';
+
```

Comment:
Use the `onReady` queue instead of adding the `app.screenSaver.initialize()` function call to the init file.
```js
app.onReady(function() {
    // Do stuff now that the app is ready.
    // This is synchronous.
});
```